src/providers/settingsProvider.js

```diff
@@ -19,6 +19,14 @@ function set(settingName, value) {
     store.set(settingName, value)
 }
 
+/**
+ * Clear the specified setting
+ * @param {string} settingName
+ */
+function clear(settingName) {
```

Comment:
is this even used?

Reply:
Nope, I think I removed what was calling it.  I'll remove this.